src/NuGet.Core/NuGet.Packaging/ContentModel/ContentPropertyDefinition.cs

```diff
@@ -144,6 +142,21 @@ public virtual bool TryLookup(string name, PatternTable table, out object value)
             return false;
         }
 
+        private static bool ContainsSlash(string name)
```

Comment:
`IndexOfAny` with a char array is a multi stage process where it first [creates a probabilistic map](https://github.com/dotnet/coreclr/blob/master/src/classlibnative/bcltype/stringnative.cpp#L480-L532) then scans for possible matches before checking for actual match. Which ends up being a little overkill for just 2 item arrays; and its faster just to check for the two chars.